src/partials/product-cards.hbs

```diff
@@ -0,0 +1,21 @@
+<div class='product-list'>
+    {{#each .}}
+    <div class='product-cards' >
+        <p class='product-name'>
+            {{name}}
+        </p>
+        <img src={{imageURL}} alt={{name}} class='product-img' />
+        <p class='product-desc'>
+            {{ description }}
+        </p>
+        <div class='product-cta-container'>
+            <p class='product-price'>
+                MRP Rs. {{ price }}
```

Comment:
Price can be inside span if you are updating it dynamically else you can remove the container.

Reply:
done